src/test/java/com/make/my/day/hm2/Homework02.java

```diff
@@ -216,19 +219,17 @@ public LazyProperty(Supplier<Integer> supplier) {
       this.supplier = supplier;
     }
 
-    //TODO: initialize "lazy" using "supplier" only one time
     public int getLazy() {
-      return 0;
+      return lazy = supplier.get();
```

Comment:
That's wrong, every time when we will use `getLazy()` it provides us new value from the supluer.

You need `initialize "lazy" using "supplier" only one time`

Reply:
Done